lib/debug/repl.cpp

```diff
@@ -196,6 +196,27 @@ std::optional<std::unique_ptr<Repl>> Repl::fromFile(const std::string &path)
     const auto bus            = accelerometer["bus"].GetUint();
     repl->addAccelerometerCommands(bus, device_address);
   }
+  if (!sensors.HasMember("temperature")) {
+    logger_.log(core::LogLevel::kFatal,
+                "Missing required field 'sensors.temperature' in configuration file");
+    return std::nullopt;
+  }
+  const auto temperature = sensors["temperature"].GetObject();
+  if (!temperature.HasMember("enabled")) {
+    logger_.log(core::LogLevel::kFatal,
+                "Missing required field 'sensors.temperature.enabled' in configuration file");
+    return std::nullopt;
+  }
+  if (temperature["enabled"].GetBool()) {
+    if (!temperature.HasMember("bus")) {
+      logger_.log(core::LogLevel::kFatal,
+                  "Missing required field 'sensors.temperature.bus' in configuration file");
+      return std::nullopt;
+    }
+    const auto device_address = temperature["device_address"].GetUint();
```

Comment:
should check if device_address exists before accessing it

Reply:
should have not simply copied